### ISSUE TITLE
Error Testing Netlify

### ISSUE THREAD

**Issue by FranprzDev** [Fundacion-Lideres-de-Ansenuza/FLA-Web#7]
### <span aria-hidden="true">❌</span> Deploy Preview for *fla-web* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 2301ec1823bb8d8bc1d4bc116133f7af53e4c047 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/fla-web/deploys/68453c2ff4affd0008c22d38 |

_Originally posted by @netlify[bot] in https://github.com/Fundacion-Lideres-de-Ansenuza/FLA-Web/issues/5#issuecomment-2953671078_
